### ISSUE TITLE
Add tour map overview page

### ISSUE THREAD

**Issue by flibbertigibbet** [azavea/cac-tripplanner#1111]
Add a map state for tour overviews. In this state, the destination is listed as the tour name, and cards are presented in the sidebar for each destination in the tour, in order. 

![image](https://user-images.githubusercontent.com/960264/65057795-2ba75900-d941-11e9-83df-c36da594b34a.png)

This is to add the map page state, with the sidebar cards; the multi-stop routing is to be implemented in #1121.

Depends on #1109.
Connects #1110 (link to this tour overview page from the tour detail page).

**Comment by flibbertigibbet** [comment]
Closed in #1148.

**Comment by KlaasH** [comment]
While working on #1146, I realized that PR #1148 doesn't actually enable the view shown in the wireframe sceenshot above.  When you click the "Map" page from a tour detail view, you get a directions page showing only the last destination on the tour, and you have to enter an origin to see the rest.  The wireframe shows a route from, I assume, the first destination to the last.

Should there be a new issue for implementing that behavior, or Is it already covered by an existing one issue, e.g. #1145?  If the latter, it might make sense to add a note to the existing issue.

**Comment by flibbertigibbet** [comment]
That is #1126, which is in progress. Events, unlike tours, are *not* to be shown with any route, but only list the destinations in the sidebar.
